examples/two_clients.rs

```diff
@@ -59,17 +60,25 @@ macro_rules! pos {
 
 fn main() {
     let _ = env_logger::init();
-    let bar_addr = Bar.listen("localhost:0".first_socket_addr(),
-                server::Options::default())
-        .wait()
-        .unwrap();
-    let baz_addr = Baz.listen("localhost:0".first_socket_addr(),
-                server::Options::default())
-        .wait()
-        .unwrap();
-
-    let bar_client = bar::SyncClient::connect(bar_addr, client::Options::default()).unwrap();
-    let baz_client = baz::SyncClient::connect(baz_addr, client::Options::default()).unwrap();
+    let mut bar_client = {
+        let reactor = reactor::Core::new().unwrap();
+        let addr = Bar.listen("localhost:0".first_socket_addr(),
+                              server::Options::from(reactor.handle()))
+                      .unwrap();
+        // TODO: Need to set up each client with its own reactor. Should it be shareable across
```

Comment:
The "need to set up" was meant to describe the status quo, not what needs to be changed.